src/components/tx/SignOrExecuteForm/index.tsx

```diff
@@ -164,7 +165,9 @@ const SignOrExecuteForm = ({
     setAdvancedParams(data)
   }
 
-  const submitDisabled = !isSubmittable || isEstimating || !tx
+  const isWrongChain = useIsWrongChain()
```

Comment:
I've just realized this branch is based off of `main` but we've recently switched to `dev` as our default branch.
Could you please rebase to dev and also change the base branch of the PR? Then this `useIsWrongChain` will be there, it's on dev.